chromebot_controller:
  type: "diff_drive_controller/DiffDriveController"
  left_wheel: ['rw_left_rotation_joint', 'fw_left_rotation_joint']
  right_wheel: ['rw_right_rotation_joint', 'fw_right_rotation_joint']
  publish_rate: 50
  pose_covariance_diagonal: [0.001, 0.001, 1.0e-3, 1.0e-3, 1.0e-3, 0.01]
  twist_covariance_diagonal: [0.001, 0.001, 1.0e-3, 1.0e-3, 1.0e-3, 0.01]


  wheel_separation: 0.3
  wheel_radius: 0.1
  cmd_vel_timeout: 0.25
  base_frame_id: base_footprint
  preserve_tuning_radius: true
  publish_cmd: false
  publish_wheel_data: true
  enable_odom_tf: true

  linear:
    x:
      has_acceleration_limits: true
      has_jerk_limits: false
      min_velocity: -0.5
      max_velocity: 1.0
      min_acceleration: -0.4
      max_acceleration: 0.8
      has_velocity_limits: true
  
  angular:
    z:
      has_velocity_limits: true
      max_velocity: 1.7
      has_acceleration_limits: true
      max_acceleration: 1.5
      has_jerk_limits: false
      
      

      
